Tasks/CopyPublishBuildArtifacts/task.json

```diff
@@ -0,0 +1,80 @@
+﻿{
```

Comment:
Why are we doing an add here of an existing task?  We're losing all history.

We should do git mv PublishBuildArtifacts CopyPublishBuildArtifacts and then edit json properties